GitUp/Application/Document.m

```diff
@@ -750,19 +750,8 @@ - (void)_updateToolBar {
 
     if ([_windowMode isEqualToString:kWindowModeString_Map]) {
       _snapshotsButton.hidden = NO;
-      if (![self validateUserInterfaceItem:(id)_snapshotsButton]) {
-        _snapshotsButton.image = [NSImage imageNamed:@"icon_nav_snapshot_disable"];
-        _snapshotsButton.alternateImage = [NSImage imageNamed:@"icon_nav_snapshot_disable"];
-        _snapshotsButton.enabled = NO;
-      } else if (_snapshotsView.superview) {
-        _snapshotsButton.image = [NSImage imageNamed:@"icon_nav_snapshot_active"];
-        _snapshotsButton.alternateImage = [NSImage imageNamed:@"icon_nav_snapshot_active_pressed"];
-        _snapshotsButton.enabled = YES;
-      } else {
-        _snapshotsButton.image = [NSImage imageNamed:@"icon_nav_snapshot"];
-        _snapshotsButton.alternateImage = [NSImage imageNamed:@"icon_nav_snapshot_pressed"];
```

Comment:
What are we losing with removing these images?

Reply:
Nothing. The snapshots button is now configured as a toggle button. With the icon set to a template, it has all the disabled, highlighted, and selected states it did before. As well, the pressed-in appearance follows the system accent color.